docs/user/updating.md

```diff
@@ -0,0 +1,25 @@
+# Updating
+
+VSCode will update the Cursorless extension automatically. In the rare case that you have extension auto-updating disabled, you will have to [update the extension manually](https://code.visualstudio.com/docs/editor/extension-marketplace#_update-an-extension-manually).
+
+## Updating the Talon side
+
+The Talon side of Cursorless always needs to be updated manually, simply by performing a `git pull`.
+
+### Linux & Mac
+
+Assuming the default install location of `~/.talon/user`:
```

Comment:
Does this sentence make sense? `~/.talon/user` is required by Talon, so can't be changed by the user.

Reply:
In my last commit, I was more clear by saying "`~/.talon/user/cursorless-talon`". Maybe this caveat isn't really necessary; I may just be biased because I have mine installed a level deeper. All of my user talon stuff is inside of a [umbrella repo](https://github.com/brollin/talon_umbrella).